### PR TITLE
Json crash on bosh validate#481

### PR DESCRIPTION
**Pull request by DarrinFong**
Changed from json.loads to simplejson.loads for more verbosity (also changed the code outside of the issue's description while we're there)

```
bosh validate invalid_json.json

[...] JSONDecodeError("Expecting ',' delimiter or '}'", s, end - 1)
[...] Expecting ',' delimiter or '}': line 9 column 2 (char 243)
```

### CODE REVIEW

**Comment by glatard** [comment]
Hi @DarrinFong, this looks great! Please check why the tests are failing and we could merge that.

File: tools/python/boutiques/publisher.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 from boutiques.logger import raise_error, print_info
 from boutiques.zenodoHelper import ZenodoError, ZenodoHelper
 import json
```
Comment (glatard) [inline tools/python/boutiques/publisher.py]: Can we remove `import json`?

File: tools/python/boutiques/tests/test_simulate.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import os
 from unittest import TestCase
 import json
```
Comment (glatard) [inline tools/python/boutiques/tests/test_simulate.py]: Same comment.

File: tools/python/boutiques/util/utils.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import os
 import json
```
Comment (glatard) [inline tools/python/boutiques/util/utils.py]: Same comment.

**Comment by DarrinFong** [comment]
Reviewing this to effectively remove all usage of json and replace with simplejson

**Comment by gkiar** [comment]
@glatard when merging this, because there was a singularity image added and removed, we should make sure to merge with a squash-commit which will just take the final changes and leave anything transient, rather than bring its bulk into history.

File: tools/python/boutiques/publisher.py (hunk 1 above)
Comment (DarrinFong) [reply tools/python/boutiques/publisher.py]: Replaces all usage of json package with simplejson

**Comment by DarrinFong** [comment]
Accidentally rebased commits that fix docker/singularity issue on Travis

**Comment by DarrinFong** [comment]
This is messy, I'll make a cleaner branch and PR that instead

**Pull request closed without merging**